gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/PermissionService.groovy

```diff
@@ -135,4 +135,8 @@ class PermissionService {
       }
     }.execute() as List<String>
   }
+
+  boolean isAdmin(String userId) {
+    return permissionEvaluator.getPermission(userId)?.isAdmin()
```

Comment:
At some point, it might make sense to actually expose an `isAdmin()` in the `FiatPermissionEvaluator` to circumvent doing this sort of thing in multiple services (I couldn't find a spot where we were already doing this from a `@PreAuthorize`).

Reply:
Yes!! Especially given the current implementation is this: https://github.com/spinnaker/fiat/blob/master/fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java#L349

Reply:
Hah, indeed ...